Library/Homebrew/os/linux/global.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
+# enables experimental readelf.rb, patchelf support.
+HOMEBREW_PATCHELF_RB = ENV["HOMEBREW_PATCHELF_RB"].present?.freeze
+
 module Homebrew
   DEFAULT_PREFIX ||= if Homebrew::EnvConfig.force_homebrew_on_linux?
     HOMEBREW_DEFAULT_PREFIX
   else
     LINUXBREW_DEFAULT_PREFIX
   end.freeze
+
+  PATCHELF_VERSION="1.0.0" if HOMEBREW_PATCHELF_RB
```

Comment:
```suggestion
  HOMEBREW_PATCHELF_VERSION = "1.0.0" if HOMEBREW_PATCHELF_RB
```

Reply:
this is no longer required thanks to bundler :sweat_smile: